docs/recipes/login.md

```diff
@@ -87,17 +95,16 @@ Our action refers renderer that can look like this:
 },
 ```
 
-## Handling callback from third-party login service
+## 2. Handling callback from third-party login service
 
-Given login was successful we should handle callback third-party service (BP Cloud in our case) performs.
-So we need to define route for it:
+After the user successfully logs in with the service provider (Botpress Cloud in this case) you need to handle the callback request on the URL provided by the actions `path` variable:
 
 ```js
 const authRouter = bp.getRouter('botpress-auth', { auth: false })
 authRouter.get('/pair/:userId', async (req, res) => {
   const user = await bp.security.getUserIdentity(req.query.identity)
 
-  // We are passing this to incoming middleware so that this can be processed as a message
+  // We are passing this to the sendIncoming middleware so that this can be processed as a message
```

Comment:
I think this should be `incoming`. `sendIncoing` is not a middleware but a function that performs this sending.

Reply:
Is the code block below this correct then?